spec/providers/firestore.spec.ts

```diff
@@ -147,8 +147,6 @@ describe('Firestore Functions', () => {
         expect(event.data.get('key1')).to.equal(true);
         expect(event.data.previous).to.not.equal(null);
         expect(event.data.previous.exists).to.be.false;
```

Comment:
If you prefer we can still keep the value of the test with something like

```typescript
// older versions throw, newer versions will return null
try {
  let val = event.data.previous.get('key1');
  expect(val).to.be.null;
  let data = event.data.previous.data;
  expect(data).to.be.null;
} catch (err) {
  // ignored; throwing is OK for older SDKs
}
```

Reply:
I'm tempted to remove them to keep things cleaner.